src/scripts/clipperUI/clipper.tsx

```diff
@@ -98,7 +99,8 @@ class ClipperClass extends ComponentBase<ClipperState, {}> {
 				isLocalFileAndNotAllowed: true,
 				selectedPageRange: "",
 				shouldAttachPdf: false,
-				shouldDistributePages: false
+				shouldDistributePages: false,
+				shouldShowPopover: false
```

Comment:
`shouldShowPopover` should also take into account allPages, which is already state. Perhaps we can narrow the scope of `shouldShowPopover` to keep it seperate?